packages/markers/src/browser/problem/problem-composite-tree-node.ts

```diff
@@ -0,0 +1,71 @@
+/********************************************************************************
+ * Copyright (C) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { CompositeTreeNode } from '@theia/core/lib/browser/tree/tree';
+import { MarkerInfoNode } from '../marker-tree';
+import URI from '@theia/core/lib/common/uri';
+import { Marker } from '../../common/marker';
+import { Diagnostic, DiagnosticSeverity } from '@theia/core/shared/vscode-languageserver-types';
+import { ProblemUtils } from './problem-utils';
+
+export interface ProblemCompositeTreeNode extends CompositeTreeNode {
+}
```

Comment:
I'm not sure whether it's necessary to define this interface, since it doesn't actually extend `CompositeTreeNode` in any concrete way?

Reply:
It wasn’t necessary, I removed it.